exercises/strings/countWords/countWords.js

```diff
@@ -7,13 +7,23 @@
  * @returns {number} The number of words in the string.
  */
 
+const countLetter = require("../countLetter/countLetter");
```

Comment:
Good job figuring out how `require` works! But remove this line if you don't use `countLetter` anywhere in this file.